alacritty/src/event.rs

```diff
@@ -714,6 +716,7 @@ impl<'a, N: Notify + 'a, T: EventListener> input::ActionContext<T> for ActionCon
                 }
 
                 self.terminal.vi_goto_point(*hint_bounds.start());
+                self.mark_dirty();
```

Comment:
Do we also need to `mark_dirty` in the `HintInternalAction::Select`, or is that handled by `update_selection`? I'd assume the latter?

Reply:
Yes, update selection does that.